### DISCUSSION TITLE
Is Demo site down?

### DISCUSSION THREAD

**Discussion by nagin** [DominoKit/domino-ui#840] [General]
Demo site https://demo.dominokit.org/home seems to be down.

**Comment by FrankHossfeld** [comment]
The demo-site has moved and I think the urls of the read me are not yet updated.

The new url for 

* v1: https://dominokit.com/domino-ui/demo/v1/home

* v2: https://dominokit.com/domino-ui/demo/v2/home

I'll think, we will see an update soon. Otherwise it might be an easy PR. :-)

**Comment by vegegoku** [comment]
> Demo site https://demo.dominokit.org/home seems to be down.

As Frank stated above we have new urls, but also as part of the migration to the new urls we setup a redirection from the old urls to the new urls, except that while we were testing the setup our domain names provider surprised us with a silly policy that blocked us from doing any further changes to our DNS for the next 7 days, and we are thinking of the migration to another DNS provider.

**Comment by FrankHossfeld** [comment]
sounds like a German one ... ;-)

**Reply by vegegoku** [reply]
GoDaddy and on top of that they dont support forwarding https domains.

**Comment by nagin** [comment]
Thanks for update!
